user_input.py

```diff
@@ -0,0 +1,4 @@
+
+
+def user_answer():
+    return input()
```

Comment:
With that kind of question, I'd rather the user not answer.. \
(I am busting your b**** here, but there is a real meaning - why do you need to wrap `input` and actually limit its usage? This extra function gives you nothing, and takes from you a lot)